### ISSUE TITLE
Adding the option to have the 'Submit' button in the questionnaire instead of outside the questionnaire

### ISSUE THREAD

**Issue by PallaviGanorkar** [google/android-fhir#1150]
**Describe the bug**
In Complete Survey flow, every form has submit button enabled as app is not allowing user to submit input with error message 'inputs are missing' though the data is field on present screen which is quit confusing for user. This behavior is observed till last form. So, submit button is expected to be disabled or submit button should not be there in UI for previous forms, it should only be present on last form where user is actually allowed to submit survey.

**Component**
SDC library, reference app

**To Reproduce**
1. Install and launch the Demo App
2. let the data download from server and patient list displayed
3. Select one patient from list where the survey is not yet completed
4. Click on 'Complete Survey' button
5. it will onpen Symptoms form for Patient Card
6. Here 'SUBMIT' button is enabled
6. Now select  option for Symptoms
7. Click 'SUBMIT' button
4. See error : 'inputs are missing'

**Expected behavior**
Submit button is expected to be disabled or submit button should not be there in UI for previous forms as user is not allowed to submit survey, it should only be present on last form where user is actually allowed to submit survey.

**Screenshots**
![SubmitButtonNotAllowToSubmit](https://user-images.githubusercontent.com/13957417/153182967-9bfcb0dd-5b78-40c6-b0a3-c50c51b30f7b.png)

**Smartphone (please complete the following information):**
 - Device:Nokia 3.4
 - Android version: 11
 - Build number: 00WW_2_410_SP01

**Comment by PallaviGanorkar** [comment]
@jingtang10  @aditya-07 

As discussed with @aditya-07 , Submit button should be removed from SDC and it will be handled in Application by application developer. But issue [1211](https://github.com/google/android-fhir/issues/1211) and PR [1201](https://github.com/google/android-fhir/pull/1201) are contradicting with the discussion.

Could you please clarify the requirement to fix this.

**Comment by jingtang10** [comment]
is this a duplicate of #1088

**Comment by jingtang10** [comment]
> @jingtang10 @aditya-07
> 
> As discussed with @aditya-07 , Submit button should be removed from SDC and it will be handled in Application by application developer. But issue [1211](https://github.com/google/android-fhir/issues/1211) and PR [1201](https://github.com/google/android-fhir/pull/1201) are contradicting with the discussion.
> 
> Could you please clarify the requirement to fix this.

discused with @PallaviGanorkar @aditya-07 and @santosh-pingle - we agreed that the option to have submit button inside the qustionnaire is useful, and santosh is working on this as part of #1088. once that change is merged, we will look at removing the submit button in the demo app so that it's using the in-questionnaire submit button. fyi @shelaghm
